Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -139,7 +139,7 @@ Note that, though it is possible to create one, having an Atom on its own\n\
 (i.e not associated with a molecule) is not particularly useful.\n";
 struct atom_wrapper {
   static void wrap() {
-    python::class_<Atom>("Atom", atomClassDoc.c_str(),
+    python::class_<Atom, Atom*>("Atom", atomClassDoc.c_str(),
```

Comment:
Was wrapping the Atom * necessary here?

Reply:
I'm assuming it was to wrap the ATOM_PTR_VECT below.

Reply:
Yeah, that's correct.